ci/test/05_before_script.sh

```diff
@@ -37,6 +37,6 @@ if [ -z "$NO_DEPENDS" ]; then
 fi
 if [ "$TEST_PREVIOUS_RELEASES" = "true" ]; then
```

Comment:
```suggestion
if [ -n "$PREVIOUS_RELEASES_TO_DOWNLOAD" ]; then
```

nit in 9226c7236496ae92bfb472470c65b95caa95ed55, because `TEST_PREVIOUS_RELEASES` is primarily used to force the test of previous releases in the test framework (and fail if not available)

Reply:
Done